Mollier/SAM.Core.Mollier.UI/Modify/AddDivisionArea.cs

```diff
@@ -0,0 +1,77 @@
+﻿using SAM.Geometry.Planar;
+using System.Windows.Forms.DataVisualization.Charting;
+using System.Collections.Generic;
+using System.Linq;
+using System;
+
+namespace SAM.Core.Mollier.UI
+{
+    public static partial class Modify
+    {
+        [Obsolete("To be changed Maciek")] // TODO: Change (MollierProcess)
+        public static void AddDivisionArea(this Chart chart, IEnumerable<UIMollierPoint> uIMollierPoints, MollierControlSettings mollierControlSettings)
+        {
+            if(mollierControlSettings.DivisionArea == false)
+            {
+                return;
+            }
+            int deltaRelativeHumidity = 10;//RH interval from neighborhoodcount
+            int deltaEnthalpy = 3;//enthalpy interval from neighborhoodcount
+            ChartType chartType = mollierControlSettings.ChartType;
+
+            //base size
+            int RH_size = 100 / deltaRelativeHumidity + 7;
+            int Ent_size = 200 / deltaEnthalpy + 7;
+
+            List<MollierPoint>[,] rectangles_points = new List<MollierPoint>[RH_size, Ent_size];//for every rh interval and every enthalpy interval it stores the list of points that belong to this area 
+            double maxCount;
+            Query.NeighborhoodCount((uIMollierPoints as List<UIMollierPoint>)?.ConvertAll(x => x.MollierPoint), out maxCount, out rectangles_points);
+
+            for (int rh = 0; rh <= 100 - deltaRelativeHumidity; rh += 10)
```

Comment:
Create predefined values for rhstart, end and step
Rename rh to relativeHumidity